src/main.rs

```diff
@@ -1,24 +1,27 @@
 /// Executable for proton_cli
-
-extern crate proton_cli;
-extern crate git2;
 extern crate rustc_serialize;
+extern crate proton_cli;
 extern crate docopt;
 
 use std::env;
 use std::path::Path;
 use docopt::Docopt;
-use proton_cli::Error;
+
+use proton_cli::error::Error;
+use proton_cli::PermissionEnum;
+use proton_cli::utils;
 
 
 const USAGE: &'static str = "
 Command-line interface for Proton
 
 Usage:
-  ./proton init <folder>
-  ./proton new-user <name> <public-key>
+  ./proton init <folder> <public-key>
+  ./proton new-user <admin-key> <name> <public-key>
+  ./proton new-sequence <admin-key> <name> <music-file>
   ./proton id-user <private-key>
-  ./proton new-sequence <name> <music-file>
+  ./proton list-permissions
+  ./proton set-permission <admin-key> (add | remove) <name> <permission> [<target>]
```

Comment:
Oh okay. I haven't actaully read that code yet. Then yeah, we should rephrase that, since admin already means something else. manager-key? Now that I'm thinking more closely about this, I think GrantPerm and EditProj could be merged in to one permission Manage. What do you think?